engine/src/main/java/org/terasology/recording/DirectionAndOriginPosRecorder.java

```diff
@@ -21,8 +21,10 @@
 import java.util.Deque;
 
 /**
- * Saves the variables 'direction' and 'originData' to be used on replays since holding the mouse buttons can cause weird
- * behavior during replays.
+ * During Record and Replay development, it was noticed that sometimes holding the right or left mouse button caused
```

Comment:
Same story plus you don't generally want to add historical information in a summary. You could have that in a separate paragraph later that goes into justification/reasoning. So a first sentence/paragraph here might be:

"Responsible for managing some variables used in a workaround during recording"

Then next sentence/paragraph you can go into the details, and perhaps what the future may hold as far as a fix that could avoid the workaround